styles.css

```diff
@@ -0,0 +1,212 @@
+* {
+	box-sizing: border-box;
+	font-family: 'Ubuntu', sans-serif;
+}
+
+body {
+  background-image: url('northern-lights.jpg');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+div.initials a {
+	display: none;
+}
+
+
+h3.name nav {
+	display: flexbox;
+	justify-content: flex-start;
+	align-content: flex-start;
+	font-size: 17px;
+	color: #16324f;
+}
+
+nav {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background-color: #9cbfc7;
+	color: #ffffff;
+	height: 50px;
+	border-radius: 15px;
+	overflow: hidden;
+}
+
+.icon img {
+	display: none;
+	width: 36px;
+}
+
+.navbar a {
+	float: left;
+	display: block;
+	text-align: center;
+	color: #16324f;
+	padding: 14px 16px;
+	text-decoration: none;
+	font-size: 17px;
+}
+
+.navbar a:hover {
+	background-color: #ddd;
+	color: #000000;
+}
+
+.navbar a.active {
+	background-color: #04aa6d;
+	color: #ffffff;
+}
+
+.main-box {
+	display: flexbox;
+	justify-content: center;
+	align-items: center;
+	position: relative;
+	left: 15%;
+	top: 30%;
+	background-color: #16324f;
+	color: #ffffff;
+    height: 75%;
+    width: 75%;
+    margin-bottom: 15px;
+	border-radius: 15px;
+	
+}
+
+header, p {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin-top: 50px;
+	padding: 20px;	
+}
+
+p {
+	display: flex;
+	align-content: center;
+	width: 80%;
+	position: relative;
+	left: 5%;
+	margin: 20px;
+}
+
+.project-section {
+	display: flex;
+	justify-content: center;
+}
+
+div.project-container {
+	display: flex;
+	justify-content: space-around;
+	flex-wrap: wrap;
+	justify-content: center;
+	width: 60%;
+	margin: 20px;
+}
+
+h2 {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.code-image img {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	width: 60%;
+	height: 60%;
+	margin-left: 45px;
```

Comment:
The image inside the project containers are not centered. Could this be fixed by using position: relative?

Reply:
Turns out it's already in my code, for some reason it didn't update on here.
This is what I found when I checked it out... .code-image img {
	display: flex;
	justify-content: center;
	align-items: center;
	position: relative;
	left: 10px;
	width: 80%;
	height: 80%;
	margin: 0 0 0 7%;
}